### DISCUSSION TITLE
Text Searching in FlowGraph

### DISCUSSION THREAD

**Discussion by Haoyu-Xiao** [Vector35/binaryninja-api#3237] [General] [answered]
`FlowGraph` is usually leveraged to show customized graphs. When the graph is large, finding interesting block becomes harder and time-wasting. However, I haven't found document about how to search texts in the `FlowGraph`. Even worse, when a `FlowGraph` is shown without saving its handle, it's hard to implement a searching interface, because there is no `current_flowgraph` variable capture the `FlowGraph` context.

**Comment by plafosse** [comment, marked as answer]
This is unfortunately a known deficiency with the existing search interfaces. We plan to address this at some point but have no firm timeline. https://github.com/Vector35/binaryninja-api/issues/2197
